scripts/release.sh

```
@@ -15,8 +15,8 @@ cd backend
 yarn version --no-git-tag-version --no-commit-hooks --no-commit --new-version $VERSION_NEW
 cd $ROOT_DIR/webapp
 yarn version --no-git-tag-version --no-commit-hooks --no-commit --new-version $VERSION_NEW
-cd $ROOT_DIR/webapp/maintenance/source
-yarn version --no-git-tag-version --no-commit-hooks --no-commit --new-version $VERSION_NEW
+cd $ROOT_DIR/maintenance
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)